apps/store/extensions/app/social-reviews/modules/review-api.js

```diff
@@ -62,7 +63,44 @@ var api = {};
         res.addHeader("Content-Type", "application/json");
         print(result);
     };
-    
+
+    var resolvePUT = function (req, res, session) {
+        var review;
+        var contentType = req.getHeader('Content-Type');
+        if (ReqUtils.parseContentType(contentType) === 'application/json') {
+            review = req.getContent();
+        }
+        else {
```

Comment:
Thanks @cnapagoda    , I have fixed it in [this commit](https://github.com/wso2/carbon-store/pull/844/commits/a937d68245d75a77dfb3df3c1fc1895ce5e1b183)